syntax = "proto3";

package protobufs.common.v1;

enum ProgrammingLanguage {
    PROGRAMMING_LANGUAGE_UNSPECIFIED = 0;
    PROGRAMMING_LANGUAGE_PYTHON = 1;
    PROGRAMMING_LANGUAGE_JAVA = 2;
    PROGRAMMING_LANGUAGE_JAVASCRIPT = 3;
    PROGRAMMING_LANGUAGE_C = 4;
    PROGRAMMING_LANGUAGE_PROLOG = 5;
}